builtin/providers/vsphere/resource_vsphere_virtual_disk_test.go

```diff
@@ -0,0 +1,125 @@
+package vsphere
```

Comment:
Looking at other tests and at helper/resource/testing.go library, it looks like negative tests aren't supported.